src/modules/lecturesight-ptzcontrol-onvif/pom.xml

```diff
@@ -59,34 +55,10 @@
 
 	<build>
     <plugins>
-
-			<!-- Generate Declarative Service Descriptor -->
-			<plugin>
-				<groupId>org.apache.felix</groupId>
-				<artifactId>maven-scr-plugin</artifactId>
-				<version>1.21.0</version>
-				<executions>
-					<execution>
-						<id>generate-scr-scrdescriptor</id>
-						<goals>
-							<goal>scr</goal>
-						</goals>
-					</execution>
-				</executions>
-				<dependencies>
-					<dependency>
-						<groupId>org.codehaus.plexus</groupId>
-						<artifactId>plexus-utils</artifactId>
-						<version>3.0.4</version>
-					</dependency>
-				</dependencies>
-			</plugin>
-
 			<plugin>
 				<groupId>org.apache.felix</groupId>
 				<artifactId>maven-bundle-plugin</artifactId>
-				<version>2.0.1</version>
-				<extensions>true</extensions>
+        <extensions>true</extensions>
```

Comment:
Indentation has gone a bit wonky in this file.

Reply:
Thanks, fixed.